cypress/integration/login.js

```diff
@@ -17,6 +17,8 @@ describe('Sourcebooks login', function() {
         cy.get('[aria-label="User"]').click();
         cy.get('[type="submit"]').click();
 
+    

```

Comment:
This is not a showstopper, but please try to remove 2 of the 3 lines with the next pull-request :)